disco/disco.go

```diff
@@ -17,7 +17,7 @@
 //
 //	messageType    byte  (the MessageType constants below)
 //	messageVersion byte  (0 for now; but always ignore bytes at the end)
-//	message-paylod [...]byte
+//	message-payload [...]byte
```

Comment:
Perhaps the other lines should be reindented... I generally try to avoid touching other lines unless I know I need to (the go linter tends to scream at me at times, and I made one change for this repo, but I think I dropped it because of the bug I mentioned earlier).